docker-image/Dockerfile

```diff
@@ -80,6 +113,21 @@ RUN cd /usr/local \
     && ./vcpkg/bootstrap-vcpkg.sh -disableMetrics \
     && chmod -R ugo+rwX /usr/local/vcpkg
 
+# RUN cd /usr/local/src \
```

Comment:
We could run these build steps (and any apt installs that are only needed for gcc) in a [stage](https://docs.docker.com/build/building/multi-stage/) and then `COPY` the install directory into the final stage of this image. This makes cleanup perfect and effortless.

Reply:
I agree that will eliminate any compilation-related bloat in the final image, if we decide to go the route of compiling the three gcc v11 or v13 suites for native, cross-arm, cross-arm64.